src/schema/basic_classes.yaml

```diff
@@ -25,14 +36,70 @@ classes:
       - description
       - alternative_identifiers
       - type
+
   OntologyClass:
+    aliases: ["OntologyClass"]
     class_uri: nmdc:OntologyClass
+    description: >-
+      This class is used to represent ontology terms.
```

Comment:
I would like to stay away form self-referential definitions. Would something like "An optimized, local representation of class defined in an external ontology" be OK?